go/arrow/array/fixed_size_list.go

```diff
@@ -207,6 +207,9 @@ func (b *FixedSizeListBuilder) Append(v bool) {
 func (b *FixedSizeListBuilder) AppendNull() {
```

Comment:
Can you just add a docstring comment here that mentions this will add the null values to the underlying value builder? Since this is a breaking change i want to make sure the documentation updates accordingly to ensure people can understand why their code broke. (if they don't read the changelog when v13 gets released haha).